pkg/virt-handler/virtwrap/libvirt/libvirt.go

```diff
@@ -50,6 +49,10 @@ type Connection interface {
 	ListSecrets() ([]string, error)
 	LookupSecretByUUIDString(uuid string) (VirSecret, error)
 	ListAllSecrets(flags libvirt.ConnectListAllSecretsFlags) ([]VirSecret, error)
+
+	// XXX: This interface is going to be removed to use
+	// virtwrap.Host
+	LookupGuestByName(name string) (VirDomain, error)
```

Comment:
Right now we have the naming convention of `VirtualMachine` on the cluster and `Domain` on the hypervisor/libvirt level. `Guest`, like you have here, might be  more descriptive than `Domain`. It would also solve the term collision with Domain as "Domain of your application".